src/components/index.tests.tsx

```diff
@@ -26,8 +33,8 @@ class IconLinkTest extends React.Component<any, any> {
         iconName="exchange"
         iconClassName="iconClassName"
         iconContainerClassName="iconContainerClassName"
-        linkClassName="linkClassName"
-        linkPosition="right"
+        textClassName="linkClassName"
```

Comment:
@roblevintennis these tests weren't failing when I committed the PR updating these fields, and running `yarn test` seemed to pass.

If these tests aren't running and preventing commits, what is the point of them? I'm finding that there's a fair bit of cognitive overhead in order to add something to EDS, and without a checklist it's becoming near-impossible for me to remember all the things to do. 😅 Not sure if this is just me, but thought I'd surface in case there are others that feel this way!

Reply:
I guess the other thing is that it's hard for both the author and the reviewer to remember all these things haha. I'm also curious how you even found this issue? I can't get an error to throw whatsoever.

Reply:
These tests are not yet automated and just confirm the typings are correct. But we could certainly benefit in having them automated. Limited bandwidth and resources on EDS is the culprit. That said, if these are things you'd like to submit a PR for it would be awesome.

Reply:
Awesome and thanks for the context! Put up #268 :)